ccdImports/configFiles/CCD_Probate_Backoffice/CaseTypeTab.json

```diff
@@ -268,3 +268,4 @@
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "Channel": "CaseWorker", "TabID": "ihtTab", "TabLabel": "IHT", "TabDisplayOrder": 18, "CaseFieldID": "deceasedHadLateSpouseOrCivilPartner", "TabFieldDisplayOrder": 6},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "Channel": "CaseWorker", "TabID": "ihtTab", "TabLabel": "IHT", "TabDisplayOrder": 18, "CaseFieldID": "ihtUnusedAllowanceClaimed", "TabFieldDisplayOrder": 7}
 ]
+
```

Comment:
just need to remove this empty line

Reply:
If the SOT tab name needs to be changed, then I think you've missed it. Tab label is still SOT

Reply:
Updated, from SOT to Legal Statement